src/api/json/catalog.json

```diff
@@ -1720,6 +1720,16 @@
       "description": "Meta-validation schema for JSON Schema Draft 4",
       "url": "http://json-schema.org/draft-04/schema"
     },
+    {
+      "name": "JSON Schema Draft 7",
+      "description": "Meta-validation schema for JSON Schema Draft 4",
```

Comment:
Looks like the version number in the description needs updating too?

Reply:
You're right. I only noticed my mistake now.